src/Cake.Npm.Tests/Ci/NpmCiToolTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using Cake.Core.Diagnostics;
+using Cake.Npm.Install;
```

Comment:
Should not be required if settings are in `Cake.Npm.Ci` namepsace

Reply:
idd